crates/core/src/geoarrow/mod.rs

```diff
@@ -87,7 +62,8 @@ impl TableBuilder {
     /// Builds a [Table]
     pub fn build(self) -> Result<Table> {
         let mut values = Vec::with_capacity(self.item_collection.items.len());
-        let mut builder = GeometryBuilder::new();
+        let geometry_type = GeometryType::new(CoordType::Interleaved, Default::default());
+        let mut builder = GeometryBuilder::new(geometry_type, false);
```

Comment:
@kylebarron got this most of the way there, the only diff I'm seeing locally is that `geoparquet::feature::tests::roundtrip` is failing because the input item has a `Polygon` but the output from `geoarrow` ends up as a `MultiPolygon`. I'm setting `prefer_multi` to `false` here ... thoughts?

Reply:
Do you have a MRE? I'll add some more tests to see if I can reproduce. What is your "output"?

Reply:
Ugh :/

Reply:
Maybe `prefer_multi` should _always_ default to `false` and users should explicitly opt-in to `true`

Reply:
Latest main should fix this.

Reply:
Worked! Thanks Kyle 🙇🏼